android/app/src/main/res/values/dimens.xml

```diff
@@ -91,4 +91,13 @@
     <dimen name="feed_item_corner_radius">3dp</dimen>
     <dimen name="feed_item_border_width">1dp</dimen>
     <dimen name="comment_list_height">300dp</dimen>
+
+    <!--Upcoming dialogbox-->
+    <dimen name="dialog_width">300dp</dimen>
+    <dimen name="upcoming_title_height">50dp</dimen>
+    <dimen name="upcoming_height">70dp</dimen>
+    <dimen name="upcoming_margin_left">30dp</dimen>
```

Comment:
@Shailesh351 @arp95 done. I have renamed some of the existing attributes to something more generic so that they can be used in future too. Is it okay now?